pkg/workloads/cortex/lib/util.py

```diff
@@ -60,8 +60,6 @@ def pp_str(obj, indent=0):
         out = json_tricks_dump(obj, sort_keys=True, indent=2)
     except:
         out = pprint.pformat(obj, width=120)
-    out = out.replace(resource_escape_seq, "@")
```

Comment:
Good catch! I decided that this should be a separate commit: https://github.com/cortexlabs/cortex/commit/7c81e819d239f0542dc7fda2da5b60defa31d5f2